src/pool.spec.ts

```diff
@@ -400,6 +439,23 @@ describe("Constant product pool", () => {
     await signAndSend(removeLiqTxGroup, account);
     await pool.updateState();
     expect(pool.state.totalLiquidity).toBe(1000);
+  }
+
+  it("Pool e2e scenario for asset with 19 decimals", async () => {
+    const testBed = await makeFreshTestBed({
+      poolType: "CONSTANT_PRODUCT",
+    });
+    e2e_scenario_19_decimals(testBed);
```

Comment:
No need to test `19_decimals` variant for old contracts.

Reply:
DONE